backend/EasySpeak.Core/EasySpeak.Core.BLL/Services/UserService.cs

```diff
@@ -21,5 +22,19 @@ public async Task<UserDto> CreateUser(UserRegisterDto userDto)
 
             return _mapper.Map<UserDto>(userEntity);
         }
+
+        public async Task<int> EnrollUserToLesson(long userId, long lessonId)
+        {
+            var user = _context.Users
+                .Include(p => p.Lessons)
+                .Single(p => p.Id == userId);
+
+            var existingLesson = _context.Lessons
+                .Single(p => p.Id == lessonId);
+
+            user.Lessons.Add(existingLesson);
+
+            return await _context.SaveChangesAsync();
```

Comment:
this is not a goof idea to load user and all their lessons for adding new lesson

Reply:
tried to refactor it but not sure..